src/components/material/digital-modal/DigitalModalBody.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { Dispatch, SetStateAction } from "react";
+import { useText } from "../../../core/utils/text";
+import TextInput from "../../atoms/input/TextInput";
+import ReservationForm from "../../reservation/forms/ReservationForm";
+
+type DigitalModalBodyProps = {
+  email: string;
+  handleOnChange: Dispatch<SetStateAction<string>>;
+  handleSubmit: () => void;
+  isLoading: boolean;
+};
+
+const DigitalModalBody: React.FunctionComponent<DigitalModalBodyProps> = ({
+  email,
+  handleOnChange,
```

Comment:
Why is the handleOnchange used as a callback here? I think it would be prettier to have a local useState for textinput changed in the same component that renders the textinput.

Reply:
I have an email state in DigitalModal because I need to send the email with the mutate function from useGetPatronInformationByPatronIdV2. and I use useGetPatronInformationByPatronIdV2 to provide the state I use to decide whether it is DigitalModalFeedback or DigitalModalBody that is rendered. so I can't move it into DigitalModalBody